miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultBarcodableViewService.java

```diff
@@ -25,7 +25,7 @@
   BarcodableViewDao barcodableViewDao;
 
   @Resource()
-  private Map<EntityType, BarcodableService> barcodableServicesMap;
+  private Map<EntityType, BarcodableService<?>> barcodableServicesMap;
```

Comment:
Is this related to the Widget documentation?

Reply:
Just a missing type parameter I noticed while going through the code related to the documentation. As there was no type before and the type I added is a wildcard, it has no effect besides clearing the warnings.

Reply:
Should these changes be moved to another PR so that the git history for these lines are clearer?

Reply:
If it were a bug fix or behaviour change, I'd agree that it deserved a separate commit, but I'd say no in this case since there is no functional change.